### PR TITLE
Full viewing range key message clarified

### PR DESCRIPTION
**Pull request by Fixer-007**
To make it sound less confusing to players. Fixes issue https://github.com/minetest/minetest/issues/6226

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -2980,8 +2980,8 @@ void Game::decreaseViewRange()
 void Game::toggleFullViewRange()
 {
 	static const wchar_t *msg[] = {
-		L"Disabled full viewing range",
-		L"Enabled full viewing range"
+		L"Normal view range",
```
Comment (HybridDog) [inline src/game.cpp]: Please also show the view range value here.

Comment (paramat) [inline src/game.cpp]: Doesn't seem essential, but i'm ok with that if other devs want it.

**Comment by paramat** [comment]
:+1:

**Comment by rubenwardy** [comment]
https://github.com/minetest/minetest/commit/e44f13aecb39e778d5b67c0894077cba2157742b

**Pull request closed without merging**